public/chat.js

```diff
@@ -10,102 +10,146 @@ $(function(){
     var user_list_area = $('#user_list')
     var room = JSON.parse(ROOM_ID)
     var user = JSON.parse(USER)
+    var debug_btn = $(".debug_btn")
     
 
     // Peer takes 1- ID, put undefined to let the server handle that
     // then { host: 3001 locally or 'your-app-name.herokuapp.com', port is either 9000 or 443(if using https)}
     // if using https include secure: true
     const myPeer = new Peer ()
+    var gblPartnerPeerId = "NoPartner"
+    var gblUsrVideoStream = ""
+    var gblUsrMediaConnection = ""
 
     myPeer.on('open', id => {
-      const peerId = id
-      socket.emit('join_room',  user ,room, socket.id ,peerId)       
+      user.peerId = id //bind a user's peerID to the user object
+      user.room = room //Room id
+      user.socketid = socket.id //socket id of the user connection
+      socket.emit('join_room',  user)       
     })
 
-    // Emit message
-    send_message.click(function() {
-        
-        socket.emit('new_chat_message', {message:message.val(), user: user.name}, room)
-        message.val('')      
-    })
-
-    //send message on press of enter inside the message box
-    message.keypress(function(e){          
-      if(e.keyCode==13){
-          send_message.click()
-      }
-    })
-
-    //Listen on new_chat_message
-    socket.on('new_chat_message', (data) => {     
-      // change formatting based on who sent the message
-      if (user.name === data.name) {
-          // show the message in the chatroom area
-          chatroom.append("<p class='chat_message myMessage'>" + data.message + "</p>")
-      } else {
-          // show the message in the chatroom area
-          chatroom.append("<p class='chat_message otherMessage'> <em>" + data.name + "</em>: " + data.message + "</p>")
-      }      
-    })
-
-
-    // get local video input
-    const myVideo = document.createElement('video')
-    myVideo.muted = true
 
+//################ PEER JS SIGNALS ########################
+    // Get local video input
+    //navigator.mediaDevices gets the media stream of the local video and/or audio stream.
     navigator.mediaDevices.getUserMedia({video:true, audio:false})
```

Comment:
Add comment to say reset to true before launch. With like (TODO) at the front, so we can search through for that before launch and change anything like this.